samples/net/openthread/coap_client_ble/src/coap_client_ble.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#include <zephyr.h>
+#include <coap_client_utils.h>
+#include <dk_buttons_and_leds.h>
+#include <logging/log.h>
+
+#include "ble_utils.h"
+
+LOG_MODULE_REGISTER(coap_client_ble, CONFIG_COAP_CLIENT_BLE_LOG_LEVEL);
+
+#define COMMAND_REQUEST_UNICAST 'u'
+#define COMMAND_REQUEST_MULTICAST 'm'
+#define COMMAND_REQUEST_PROVISIONING 'p'
+
+#define CON_STATUS_LED DK_LED2
+
+#define RUN_STATUS_LED DK_LED1
+#define RUN_LED_BLINK_INTERVAL 1000
+
+static void on_nus_received(struct bt_conn *conn, const u8_t *const data,
+			    u16_t len)
+{
+	LOG_INF("Received data: %s", log_strdup(data));
+
+	if (len != 1) {
```

Comment:
not sure how this nus works, but is it possible to get multiple commands packed together  e.g. when multiple commands are issued before single timeslot?

Reply:
This is not our scenario. Our API are single characters: 'u', 'm', 'p' so we check this. NUS operates C strings so that we can get e. g. "mpu" but this is not what here is about.